mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -275,6 +321,13 @@ class GBTClassificationModel private[ml](
   @Since("2.0.0")
   lazy val featureImportances: Vector = TreeEnsembleModel.featureImportances(trees, numFeatures)
 
+  private def classProbability(loss: String, rawPrediction: Double): Double = {
```

Comment:
adding  "private val oldLoss: ClassificationLoss = getOldLossType" won't work because getOldLossType returns a Loss and not a LogLoss, which doesn't have computeProbability.  However, I did add the ClassificationLoss trait and in ClassProbability I just call LogLoss.computeProbability.  I'm not sure if it will pass the binary compat checks though, let's see...